### DISCUSSION TITLE
Langue occitane

### DISCUSSION THREAD

**Discussion by adminieo** [organicmaps/organicmaps#4892] [Ideas]
Bonjour. Nous avons traduit l'interface d'OsmAnd en occitan et cette application utilise la carte Osm-oc.

Je me propose de localiser l’interface en occitan.
Pouvez-vous étudier la possibilité de mettre le fond de carte Osm en occitan ? 

Merci Projet IEO-Geoccitania

**Comment by Kazing** [comment]
Bonjour,
Pour la traduction de l'interface, vous trouverez les infos nécessaires pour contribuer ici : https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md
N'hésitez pas si il y a des questions

Une fois que la langue est ajouté dans l'appli, le fond de carte peut afficher les noms des lieux dans la langue sélectionné par l'utilisateur, donc si l'utilisateur sélectionne l'occitan et si `name:oc` est présent dans OSM, c'est le nom occitan qui sera affiché.

**Comment by adminieo** [comment]
Merci de vos explications. Je vais voir tout ça. Pèire Brechet www.openstreetmap-oc.orgMandat de mon iFònLe 3 avr. 2023 à 13:55, Kazing ***@***.***> a écrit :﻿
Bonjour,
Pour la traduction de l'interface, vous trouverez les infos nécessaires pour contribuer ici : https://github.com/organicmaps/organicmaps/blob/master/docs/TRANSLATIONS.md
N'hésitez pas si il y a des questions
Une fois que la langue est ajouté dans l'appli, le fond de carte peut afficher les noms des lieux dans la langue sélectionné par l'utilisateur, donc si l'utilisateur sélectionne l'occitan et si name:oc est présent dans OSM, c'est le nom occitan qui sera affiché.
